fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -77,40 +68,21 @@ public RootRdfContext(final FedoraResource resource,
 
     private void concatRepositoryTriples() {
         LOGGER.trace("Creating RDF triples for repository description");
-        final Repository repository = session().getRepository();
 
         final ImmutableSet.Builder<Triple> b = builder();
-        stream(repository.getDescriptorKeys()).forEach(key -> {
-            final String descriptor = repository.getDescriptor(key);
-            if (descriptor != null) {
-                // Create a URI from the jcr.Repository constant values,
-                // converting them from dot notation (identifier.stability)
-                // to the camel case that is more common in RDF properties.
-                final String uri = stream(key.split("\\."))
-                        .map(StringUtils::capitalize).collect(joining("", REPOSITORY_NAMESPACE + "repository", ""));
-                b.add(create(subject(), createURI(uri), createLiteral(descriptor)));
-            }
-        });
 
         /*
             FIXME: removing due to performance problems, esp. w/ many files on federated filesystem
             see: https://www.pivotaltracker.com/story/show/78647248
 
+            final Repository repository = session().getRepository();
+
             b.add(create(subject(), HAS_OBJECT_COUNT.asNode(), createLiteral(String
                     .valueOf(getRepositoryCount(repository)))));
             b.add(create(subject(), HAS_OBJECT_SIZE.asNode(), createLiteral(String
                     .valueOf(getRepositorySize(repository)))));
         */
 
-        // Get the cluster configuration, if available
-        // this ugly test checks to see whether this is an ordinary JCR
-        // repository or a ModeShape repo, which will possess the extra info
-        if (JcrRepository.class.isAssignableFrom(repository.getClass())) {
```

Comment:
I believe this cluster information is still valuable... especially with the recent flurry of interest in clustering.